tools/statistical_ensemble_test/single_run.py

```diff
@@ -234,84 +256,100 @@ def single_case(opts_dict, case_flags, stat_dir):
     ret = os.system(command)
 
     #time steps
-    if opts_dict['uf'] == True:
-        command = './xmlchange --file env_run.xml --id STOP_OPTION --val nsteps'
-        ret = os.system(command)
-        command = '  ./xmlchange --file env_run.xml --id STOP_N --val 9'
-        ret = os.system(command)
+    if opts_dict['ect'] == 'pop':
```

Comment:
We are using python - I think these should be changed from xmlchange commands to directly access the case object.

Reply:
OK - will change in future PR